src/main/java/com/razorpay/CustomerClient.java

```diff
@@ -22,6 +22,14 @@ public Customer edit(String id, JSONObject request) throws RazorpayException {
     return put(String.format(Constants.CUSTOMER_EDIT, id), request);
   }
 
+  public List<Customer> fetchAll() throws RazorpayException {
+    return fetchAll(null);
```

Comment:
why sending null?

Reply:
because in other api they also sending null param . when i am trying to remove null it shows `Exception in thread "main" java.lang.StackOverflowError`